static/app/views/issueDetails/streamline/eventTitle.tsx

```diff
@@ -202,68 +168,12 @@ export function EventTitle({event, group, ref, ...props}: EventNavigationProps)
             </Fragment>
           )}
         </EventInfo>
-        {eventSectionConfigs.length > 0 && (
-          <JumpTo>
```

Comment:
found it a bit easier to understand by moving the logic for the jump section into its own component